### PR TITLE
[MIG] [18.0] project_team: migrated module

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Jayram

### CODE REVIEW

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

File: project_team/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     'summary': 'Project Team Management',
     'category': 'Project Management',
     'website': 'https://www.serpentcs.com',
-    'version': '17.0.1.0.0',
+    'version': '18.0.1.0.0',
     'license': 'AGPL-3',
```
Comment (Trivedi-Vacha-SerpentCS) [inline project_team/__manifest__.py]:
```suggestion
    'license': 'LGPL-3',
```

**Comment by ketanserpentcs** [comment]
Testing is assigned to devraj

**Comment by ketanserpentcs** [comment]
QA done with index by Devraj

**Comment by Trivedi-Vacha-SerpentCS** [comment]
@ShwetaSerpentCS kindly verify index and banner

**Pull request merged**